Automattic/woocommerce-payments
# Add support for Romania

Fixes #5974 

#### Changes proposed in this Pull Request


Add RON currency support.







#### Testing instructions





* On server side checkout 3264-gh-Automattic/woocommerce-payments-server branch

* **Base fees**
* Go to Woo -> Settings -> General and set Store Country to Romania and currency to Romanian Leu
* Remove supported countries from the account on server side
* Place an order - base fees should be applied
<img width="1000" alt="Screenshot 2023-04-18 at 18 16 54" src="https://user-images.githubusercontent.com/82826872/232823315-c4e4f691-8921-4008-9bb8-10cbdda3f58b.png">

* **FX fees**
* change store currency to EUR
* Remove supported countries from the account on server side
* Place an order - base and FX fees should be applied
<img width="1004" alt="Screenshot 2023-04-18 at 18 19 45" src="https://user-images.githubusercontent.com/82826872/232824070-21146534-fd7c-4fee-8800-7427586dff76.png">

* **EPS fees**
* Place an order with EPS 
* You will be redirected to a Stripe test page where you can approve/fail the order
* Approve the order - EPS fees should be applied
<img width="1039" alt="Screenshot 2023-04-18 at 18 29 39" src="https://user-images.githubusercontent.com/82826872/232826922-62d9512a-f349-4312-8da8-cf8416929bb8.png">

Continue with all fees defined for Romania.

-------------------


**Post merge**
